servant-client/servant-client.cabal

```diff
@@ -46,9 +56,14 @@ library
     , text
     , transformers
     , transformers-compat
+    , case-insensitive
```

Comment:
Turned out to be a bit more complicated: https://github.com/haskell-servant/servant/commit/d2d25954c3bbd04a9bddadf8d3abd8a57488aa33.
Now the test-suite doesn't depend on the library but uses the files from `src` directly. I don't know how to do it differently without putting `parseHeaders` into the public api.